misc/scripts/models-as-data/bulk_generate_mad.py

```diff
@@ -36,8 +39,23 @@ class Project(TypedDict):
     """
 
     name: str
-    git_repo: str
+    git_repo: NotRequired[str]
     git_tag: NotRequired[str]
+    with_sinks: NotRequired[bool]
+    with_sinks: NotRequired[bool]
+    with_summaries: NotRequired[bool]
+
+
+def shouldGenerateSinks(project: Project) -> bool:
```

Comment:
I think the the Python naming convention is
```suggestion
def should_generate_sinks(project: Project) -> bool:
```
That also applies to the two functions below.